### ISSUE TITLE
Consider more specific error message when incorrect value is entered for epidatr

### ISSUE THREAD

**Issue by tinatownes** [cmu-delphi/epidatr#279]
When using the epidatr API, if an incorrect value is entered in an attempt to fetch data for a signal, the only message received is:
![image](https://github.com/cmu-delphi/delphi-epidata/assets/11445176/cc9c358e-a7e5-43a2-9d07-47facbe82409)
Is it possible to have a more descriptive warning message that informs user what actual value is incorrect?

In this example, source = "fb-survey" is incorrectly written as "fb_survey", but the warning message does not specify what value is problematic:
![image](https://github.com/cmu-delphi/delphi-epidata/assets/11445176/55961869-e1e6-4d3f-8095-075c31eb3184)

**Comment by dshemetov** [comment]
Agreed, that error message is not very helpful and the source + signal names are very fragile. This should be quite doable, especially given that we already have code that parses metadata and builds a table of available source-signal pairs.
